app/javascript/components/workflows/professor/exams/admin.tsx

```diff
@@ -28,6 +26,9 @@ import ReadableDate from '@hourglass/common/ReadableDate';
 import { hitApi } from '@hourglass/common/types/api';
 import { AlertContext } from '@hourglass/common/alerts';
 import DateTimePicker from '@professor/exams/new/DateTimePicker';
+import createVersion from '@hourglass/common/api/professor/exams/versions/create';
+import deleteVersion from '@hourglass/common/api/professor/exams/versions/delete';
+import TooltipButton from '@hourglass/workflows/student/exams/show/components/TooltipButton';
```

Comment:
Minor cosmetic nit: we have `@student` `@professor` etc for these; we should avoid having `@hourglass/workflows/` when not needed

Reply:
Yeah, that was the auto-importer doing its thing.